modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/facet/display/builder/AssetCategoriesSearchFacetDisplayBuilder.java

```diff
@@ -250,6 +255,20 @@ public void setParameterValues(String... parameterValues) {
 		return buckets;
 	}
 
+	protected AssetCategoriesSearchFacetDisplayContext
+		createAssetCategoriesSearchFacetDisplayContext(
+			RenderRequest renderRequest) {
+
+		try {
+			return new AssetCategoriesSearchFacetDisplayContext(
+				com.liferay.portal.kernel.util.PortalUtil.getHttpServletRequest(
```

Comment:
1. Is there a reason why we're using the full package path `com.liferay.portal.kernel.util.PortalUtil` rather than importing `PortalUtil`?
2. Instead of using `PortalUtil` let's add `Portal portal` to the constructor and call `_portal.getHttpServletRequest` directly. We should try to avoid using these `*Util` classes that basically redirect to another method.